packages/interactivity/src/hooks.tsx

```diff
@@ -268,7 +204,7 @@ const resolve = ( path, namespace ) => {
 };
 
 // Generate the evaluate function.
-export const getEvaluate: GetEvaluate =
+export const getEvaluate: any =
```

Comment:
Using GetEvaluate is returning this error:
```
 Evaluate; ref: RefObject<HTMLElement>; attributes: createElement.JSX.HTMLAttributes<EventTarget>; }' is not assignable to type 'Scope'.
  Types of property 'context' are incompatible.
    Type 'DeepSignalObject<{}>' is missing the following properties from type 'Context<any>': Consumer, Provider
407  ? getEvaluate( { scope } )( eachKey[ 0 ] )
```

Reply:
I'll take a look, deep signal types can be complex 🙂